src/tsickle.ts

```diff
@@ -923,33 +923,20 @@ class Annotator extends ClosureRewriter {
 
     // Gather the names of local exports, to avoid reexporting any
     // names that are already locally exported.
-    // To find symbols declared like
-    //   export {foo} from ...
-    // we must also query for "Alias", but that unfortunately also brings in
-    //   import {foo} from ...
-    // so the latter is filtered below.
-    const locals =
-        this.typeChecker.getSymbolsInScope(this.file, ts.SymbolFlags.Export | ts.SymbolFlags.Alias);
-    const localSet = new Set<string>();
-    for (const local of locals) {
-      if (local.declarations &&
-          local.declarations.some(d => d.kind === ts.SyntaxKind.ImportSpecifier)) {
-        continue;
-      }
-      localSet.add(local.name);
-    }
-
+    const moduleSymbol = this.typeChecker.getSymbolAtLocation(this.file);
+    const moduleExports = moduleSymbol && moduleSymbol.exports || new Map<string, ts.Symbol>();
```

Comment:
It is not - it selects the new map if `exports` is undefined, in addition to `moduleSymbol` being falsy.